catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/requests/SubmitStandingQueryRequestImpl.java

```diff
@@ -433,39 +473,57 @@ public void run() {
                 "Start time for subscription is in the future, waiting {} seconds", waitSecs);
             try {
               Thread.sleep(waitToStart);
-            } catch (Exception ignore) {
+            } catch (InterruptedException exception) {
+              // if we were interrupted, break out to allow exit
```

Comment:
the break is commented out, will we do it?

Reply:
Nope. Added a `Thread.currentThread().interrupt();` instead